### PR TITLE
Update draft-saxe-wimse-token-exchange-and-translation-protocol.md

### PR DESCRIPTION
**Pull request by dhs-aws**
Fixing normative references

### CODE REVIEW

**Review by gffletch — CHANGES_REQUESTED** [review]
I fixed this issue in my PR. We can delete this one :)

File: draft-saxe-wimse-token-exchange-and-translation-protocol.md [hunk 1]
```diff
@@ -33,13 +33,14 @@ author:
...
...ed for OAuth 2.0 in {{RFC8693}}, allowing the exchange of access tokens, refresh tokens, OpenID Connect ID Token (OIDC), and SAML assertions for new OAuth access tokens. However, for workloads, there exist a broad array of input and output token types which must be considered beyond the input types supported by {{RFC8693}}.  These token types include, but are not limited to, SPIFFE SVIDs, x.509 certificates, Amazon sigv4A, macaroons, <...>.  Further, these tokens may be encoded in formats including JWT, CBOR, and protocol buffers (protobufs).  Given the variety and complexity of input and output token types and encoding, a strict token exchange that maintains all of the contextual information from the input token to the output token may not be possible.  We define these non-RFC8693 use cases with potentially lossy conversions as "token translation" (e.g. information may be lost in translation).   In this document we describe a workload profile for token exchange, using the mechanisms in {{RFC8693}}, and a new set of translations between arbitrary token types.  Additionally, we define mechanisms to enrich tokens during translation to support the use cases defined in <Use Cases Doc>.
```
Comment (adeinega) [inline draft-saxe-wimse-token-exchange-and-translation-protocol.md:43]: I suggest replacing `Token exchange is well defined for OAuth 2.0 in {{RFC8693}}` by "Token exchange is introduced in {{RFC8693}}".

IMO, RFC8693 is a bit abstract, it's barely possible to use before you gets into details of an AS's implementation you deal with.

Comment (adeinega) [reply draft-saxe-wimse-token-exchange-and-translation-protocol.md:43]: Kindly ask to mention Kerberos service tickets, there seem to be an interest to use them.

Comment (adeinega) [reply draft-saxe-wimse-token-exchange-and-translation-protocol.md:43]: For CBOR, we got CWTs, [RFC 8392](https://datatracker.ietf.org/doc/html/rfc8392).

I suggest mentioning them in `these token types include, but are not limited to ...` with a link to their RFC, and accordingly omitting this format in the second sentence.

The same applies to JWTs, I encourage our WG to be very precise that we would like to deal with JWT ATs, no with all JWTs out there due to the security concerns (I'll mention them separately).

Comment (adeinega) [reply draft-saxe-wimse-token-exchange-and-translation-protocol.md:43]: By JWT ATs, I mean [JSON Web Token (JWT) Profile for OAuth 2.0 Access Tokens (RFC 9068)](https://www.rfc-editor.org/rfc/rfc9068.html).

Comment (adeinega) [reply draft-saxe-wimse-token-exchange-and-translation-protocol.md:43]: https://openid.net/ doesn't use term "OIDC" as I recall.

Comment (dhs-aws) [reply draft-saxe-wimse-token-exchange-and-translation-protocol.md:69]: I'm not sure what the request is for OIDC.  Leaving it for now, we can update as we continue writing.

Comment (adeinega) [reply draft-saxe-wimse-token-exchange-and-translation-protocol.md]: @dhs-aws, my suggestion was to replace "OIDC" by "OpenID Connect", that's it. https://openid.net/ did use this abbreviation in a couple of places only, say in the names of some they specs such as OIDC4VP and OIDC4VCI but they don't do it any longer.

**Pull request merged**